modules/administration-guide/partials/assembly_backing-up-a-che-instance-to-amazon-s3-by-using-prod-cli.adoc

```diff
@@ -0,0 +1,16 @@
+:parent-context-of-backing-up-a-{prod-id-short}-instance-to-amazon-s3-by-using-prod-cli: {context}
+
+[id="backing-up-a-{prod-id-short}-instance-to-amazon-s3-by-using-prod-cli_{context}"]
+= Backing up a {prod-short} instance to Amazon S3 by using {prod-cli} 
+
+:context: backing-up-a-{prod-id-short}-instance-to-amazon-s3-by-using-prod-cli
+
+You can use {prod-cli} to make a backup snapshot of a {prod-short} instance and send the snapshot to the configured Amazon S3. To do so, enter {prod-cli} with the command-line options or set the environment variables.
```

Comment:
```suggestion
You can use {prod-cli} to make a backup snapshot of a {prod-short} instance and send the snapshot to the configured Amazon S3 or API compatible storage server. To do so, enter {prod-cli} with the command-line options or set the environment variables.
```

Reply:
@mmorhun, I added `or S3 API compatible storage`, which is the term I used on another page.